front/temporal/upsert_queue/activities.ts

```diff
@@ -146,3 +174,107 @@ export async function upsertDocumentActivity(
     upsertContext: upsertQueueItem.upsertContext || undefined,
   });
 }
+
+async function upsertTableItem(
```

Comment:
Can we move to a dedicated workflow and queue to avoid impacting regular upserts?

Reply:
As per IRL, dedicated workflow :+1: 
The dedicated queue is for another time